Rubberduck.Core/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -412,7 +412,8 @@
                   ItemContainerStyle="{StaticResource TreeViewContainerStyle}"
                   FontSize="{Binding FontSize, Mode=OneWay}"
                   BorderThickness="0,1"
-                  VirtualizingPanel.IsVirtualizing="False">
+                  VirtualizingPanel.IsVirtualizing="False"
+                  KeyDown="ProjectTree_KeyDown">
```

Comment:
How about:

```xml
<TreeView.InputBindings>
    <KeyBinding Key="Delete" Command="{Binding RemoveCommand}" />
</TreeView.InputBindings>
```

Reply:
Binding to a key doesn't have [CommandBase.Execute](https://github.com/rubberduck-vba/Rubberduck/blob/b59352933cbf346f096921bc254cefdff8710719/Rubberduck.Core/UI/Command/CommandBase.cs#L56) fire when the key is pressed. I tested with home/end to go to the top/bottom. Attached those to the [DeleteComand](https://github.com/rubberduck-vba/Rubberduck/blob/b59352933cbf346f096921bc254cefdff8710719/Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerViewModel.cs#L389) and nothing occurred when either was pressed. My guess is something is intercepting it. Puzzling through it.

Reply:
Aw, too bad. Merging as-is then.